### PR TITLE
Rename app to 'BitriseReactNativeSample'

### PR DESCRIPTION
**Pull request by hisaac**
This PR renames the app from ReactNativeSampleApp to BitriseReactNativeSample.

### CODE REVIEW

File: android/app/src/debug/java/com/reactnativesampleapp/ReactNativeFlipper.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
  * directory of this source tree.
  */
-package com.reactnativesampleapp;
+package com.BitriseReactNativeSample;
```
Comment (jakerobb) [inline android/app/src/debug/java/com/reactnativesampleapp/ReactNativeFlipper.java]: I would be inclined to keep the package name all lowercase; that's the Java standard.

File: ios/ReactNativeSampleApp.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:ReactNativeSampleApp.xcodeproj">
+      location = "group:BitriseReactNativeSample.xcodeproj">
```
Comment (jakerobb) [inline ios/ReactNativeSampleApp.xcworkspace/contents.xcworkspacedata:5]: Unrelated to this change: I don't know this for sure, but this seems like a file that should be `.gitignore`d. WDYT?

**Review by jakerobb — COMMENTED** [review]
A couple thoughts here.

File: ios/ReactNativeSampleApp.xcworkspace/contents.xcworkspacedata (hunk 2 above)
Comment (hisaac) [reply ios/ReactNativeSampleApp.xcworkspace/contents.xcworkspacedata:5]: Fair question. This file defines how Xcode opens and handles the project, so we do want it to be checked in.

File: android/app/src/debug/java/com/reactnativesampleapp/ReactNativeFlipper.java (hunk 1 above)
Comment (hisaac) [reply android/app/src/debug/java/com/reactnativesampleapp/ReactNativeFlipper.java]: Good catch! I'll update that.

- [x] Lowercase the package name in the Android app

Comment (hisaac) [reply android/app/src/debug/java/com/reactnativesampleapp/ReactNativeFlipper.java]: Updated in https://github.com/bitrise-io/getting-started-react-native-sample-app/pull/3/commits/6b3fd59c95779d431efb41568e3694da22f0e72f

**Review by jakerobb — APPROVED** [review]
LGTM! :)

**Pull request merged**